src/classkit.cs

```diff
@@ -186,6 +186,11 @@ interface CLSActivity {
 
 		[Export ("stop")]
 		void Stop ();
+
+		[Introduced (PlatformName.MacCatalyst, 14,5)]
```

Comment:
You can use the short form `[MacCatalyst (14, 5)]`

Reply:
it's a copy/paste from sharpie - making it _kind of_ shorter 😄 
but worth adding to sharpie, c.c. @dalexsoto

Reply:
Issue'ified => https://github.com/xamarin/ObjectiveSharpie/issues/143 

I can forget 🐟🧠 but GitHub will not 🐘